commands/job.go

```diff
@@ -22,60 +22,93 @@ var JobCmd = &cli.Command{
 }
 
 type jobControlOpts struct {
-	ID int
+	ID      int
+	apiAddr string
 }
 
 var jobControlFlag jobControlOpts
 
 var JobStartCmd = &cli.Command{
-	Name:   "start",
-	Usage:  "start a job.",
-	Before: initialize,
-	After:  destroy,
+	Name:  "start",
+	Usage: "start a job.",
 	Flags: []cli.Flag{
 		&cli.IntFlag{
 			Name:        "ID",
 			Usage:       "ID of job to start",
 			Required:    true,
 			Destination: &jobControlFlag.ID,
 		},
+		&cli.StringFlag{
+			Name:        "api",
```

Comment:
An alternative to this is getting the apiAddr from the repo (we set it [here](https://github.com/filecoin-project/sentinel-visor/blob/lily/commands/daemon.go#L57)), similar to what's done here: 
https://github.com/filecoin-project/lotus/blob/c41777dcd2aea4e32405b7c1f548c7621cd97cd4/cli/util/api.go#L110-L133

Reply:
That wouldn't work if the client was on a different computer though.